### PR TITLE
ISSUE-109-Artem-Grachev

### PR DESCRIPTION
**Pull request by ArtyomGrachev**
Review it pls

### CODE REVIEW

File: tasks [hunk 1]
```diff
@@ -86,7 +86,7 @@
 55 + 95
 94 + 55
 29 + 49 = 78
-68 - 27 = 38
+68 - 27 = 41
```
Comment (ArtyomGrachev) [inline tasks]: Error Solved

**Comment by ArtyomGrachev** [comment]
Reviewed myself, cause it's kinda hard to find an external reviewer at 2 a.m.

File: tasks [hunk 2]
```diff
@@ -85,9 +85,9 @@
 76 - 72
 90 - 86
 55 + 95
-94 + 44
-29 + 49
-68 - 27
+94 + 55
+29 + 49 = 78
+68 - 27 = 38
```
Comment (SvetJul) [inline tasks]: I believe there is a mistake here.

**Comment by ArtyomGrachev** [comment]
> Please, fix the error.

Error was fixed

**Comment by dehasi** [comment]
`ISSUE-109-Artem-Grachev` => `ISSUE-109 Artem Grachev `

**Pull request merged**